app/screens/image_preview/downloader.android.js

```diff
@@ -127,7 +123,7 @@ export default class Downloader extends PureComponent {
                         useDownloadManager: true,
                         notification: true,
                         path: dest,
-                        title: `${file.caption} ${title}`,
+                        title: `${file.caption}`,
```

Comment:
So the title is causing the destination file to have the suffix? I'm not really following

Reply:
I think you can look at https://github.com/joltup/rn-fetch-blob/wiki/Classes#rnfetchblobandroiddownloadoption . You can see that the `title` is the title of the file displayed in the `downloads` folder.

Reply:
awesome!!! 🎉